luthier/config/src/dimensions.lua

```diff
@@ -69,7 +69,8 @@ local field = dimensionUtils.field
 local FIELDSETS = {
     default = { pk "ID", field "DESC" },
     country = { pk "ID", field("DESC", "COUNTY", "STATE")},
-    page = { pk "ID", field "DESC" }
+    page = { pk "ID", field "DESC" },
+    test = { pk "TEST_PK", field ("TEST_FIELD_1", "TEST_FIELD_2", "TEST_FIELD_3") }
```

Comment:
There shouldn't be a space between `field` and `("TEST_FIELD_1", ...)`. It's a
function call.

Reply:
I was not aware of that. Thanks for pointing it out.